Mathlib/LinearAlgebra/PerfectPairing.lean

```diff
@@ -45,6 +49,34 @@ variable {R M N}
 
 namespace PerfectPairing
 
+/-- If the coefficients are a field, and one of the spaces is finite-dimensional, it is sufficient
+to check only injectivity instead of bijectivity of the bilinear form. -/
+def mkOfFiniteDimensional {K V W : Type*}
```

Comment:
Hmm, I'm wondering whether
```suggestion
def mkOfInjective {K V W : Type*}
```
might fit a tiny bit better. Do you have a preference?

Reply:
Oh yeah, this is a much better name; thanks.